packages/itwinui-css/src/surface/surface.scss

```diff
@@ -2,19 +2,19 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 
 @mixin iui-surface(
-  $background-color: var(--iui-color-background-1),
+  $background-color: var(--iui-color-background),
+  $border-color: var(--iui-color-border-subtle),
```

Comment:
Does `@mixin iui-surface` need to match the React surface component?  I think `@mixin iui-surface` should provide the best defaults for use within iTwinUI-CSS so we don't need to specify elevation in most cases, even if that is different from the React Surface component's defaults.

Reply:
It doesn't need to match, but I'm just saying that these changes *do* affect Surface, meaning we need to override them in react.

Maybe there needs to be a different mixin (or a predetermined set of default values) for the Surface component.

The distinction between "internal surface mixin" and "public Surface component" is useful, since the latter is meant to be used more like "cards" where the users can put other components inside.

Reply:
> The distinction between "internal surface mixin" and "public Surface component" is useful, since the latter is meant to be used more like "cards" where the users can put other components inside.

This makes sense to me.

Reply:
Introduced `@mixin iui-surface-component` which is used by `.iui-surface`.  We have a separate `@mixin iui-surface` which has had `border-radius` changed to simplify the general usage of surface across components.

Reply:
Issue created #798.